lib/thrift/protocols/binary.ex

```diff
@@ -53,10 +49,9 @@ defmodule Thrift.Protocols.Binary do
   def serialize(_, nil) do
     []
   end
-  def serialize(:bool, value) do
-    value = bool_to_int(value)
-    <<value::8-signed>>
-  end
+  def serialize(:bool, nil),   do: <<0::8-signed>>
```

Comment:
I think implicit here is whether we want to watch to support "truthy" / "falsey" inputs or if we should explicitly require `true` or `false` atoms (where all other inputs result in an `:error`).